docs/chainlink-keepers/cron-contracts.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: nodes.liquid
+section: ethereum
+date: Last Modified
+title: 'Keepers Job Scheduler [BETA]'
+whatsnext:
+  {
+    'FAQs': '/docs/chainlink-keepers/faqs/',
+  }
+---
+
+> 👍 Beta
+>
+> The Keepers Job Scheduler is currently in Beta.
+
+# Overview
+In this tutorial you will use the Chainlink Keepers Job Scheduler to set up a time schedule to automatically execute smart contract functions.
```

Comment:
```suggestion
In this tutorial you will use the Chainlink Keepers Job Scheduler to automatically execute smart contract functions on a specific time schedule.
```
This might read more smoothly? Feel free to reject this suggestion.